new/analyse/create-package-json-content/get-package-json.ts

```diff
@@ -0,0 +1,64 @@
+import { Env, PackageAfterCreatePackageJsonContent } from '../../types';
+
+import { clone } from 'ramda';
+
+function getPackageJson(
+  packages: Env['packages'],
+  packageName: string,
+  type: 'nexus'
+): PackageAfterCreatePackageJsonContent['packageJsonNexus'];
+function getPackageJson(
+  packages: Env['packages'],
+  packageName: string,
+  type: 'git'
+): PackageAfterCreatePackageJsonContent['packageJsonGit'];
+function getPackageJson(
+  packages: Env['packages'],
+  packageName: string,
+  type: 'nexus' | 'git'
+):
+  | PackageAfterCreatePackageJsonContent['packageJsonNexus']
+  | PackageAfterCreatePackageJsonContent['packageJsonGit'] {
+  const isNexus = type === 'nexus';
+
+  const clonePackageJson = clone(packages![packageName].packageJson);
+  const dependsOn = isNexus
+    ? Object.fromEntries(
+        packages![packageName].dependsOn.map((dependency) => [
+          dependency.name,
+          dependency,
+        ])
+      )
+    : {};
+
+  // package has dependencies
+  if (clonePackageJson.dependencies) {
+    // set all dependencies packages' versions as "*"
+
+    clonePackageJson.dependencies = Object.fromEntries(
+      Object.keys(clonePackageJson.dependencies)
+        .filter((dependencyName) => dependencyName in packages!)
+        .map((dependencyName) => [
+          dependencyName,
+          isNexus ? dependsOn[dependencyName].range : '*',
+        ])
+    );
+  }
+
+  // package has peerDependencies
+  if (clonePackageJson.peerDependencies) {
+    // set all peerDependencies packages' versions as "*"
+    clonePackageJson.dependencies = Object.fromEntries(
+      Object.keys(clonePackageJson.peerDependencies)
+        .filter((dependencyName) => dependencyName in packages!)
+        .map((dependencyName) => [
+          dependencyName,
+          isNexus ? dependsOn[dependencyName].range : '*',
+        ])
+    );
+  }
+
```

Comment:
We should set all dev dependencies to `*` here (for both cases). They're always `*` also in case someone writes stuff into the dependency range.

Reply:
I'll add them